examples/php-fpm/php/run.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+php-fpm -D
+
+screen -d -m sh -c "SCRIPT_FILENAME=./index.php REQUEST_METHOD=GET cgi-fcgi -bind -connect 127.0.0.1:9000"
```

Comment:
This should work fine without `screen`, it'd be nice to avoid the dependency when running the example :)

Reply:
done